docs/external-plugin-messaging.md

```diff
@@ -0,0 +1,87 @@
+# External Plugin Messaging
+
+Other Plugin Hub plugins can publish a `PluginMessage` via `EventBus#post` that instructs Dink to submit a webhook request.
+
+Users can opt-out of this capability by disabling `External Plugin Requests > Enable External Plugin Notifications`.
+
+Plugins can request that a screenshot is included with the notification, but users can also opt-out by
+disabling `External Plugin Requests > Send Image` (default: on) and `External Plugin Requests > Allow Overriding 'Send Image'` (default: off).
+
+Plugins can include a Discord url for the webhook, otherwise Dink will utilize `External Webhook Override`
+(or `Primary Webhook URLs` if an external url override is not specified).
+If a plugin requests a non-Discord url, it will be ignored in favor of the Dink configuration.
+
+Otherwise, external plugins have a great deal of customization for the webhook body.
```

Comment:
remove this line or change it to something like "Here's how plugins can customize the webhook body"